int16[11] angle # unit: [0.1 deg]
uint16[3] led # r, g, b 
int16 time # unit: [16.667 msec]
